app/routeMap.js

```diff
@@ -13,6 +13,7 @@ let discourse = require('./discourse')
 
 module.exports = {
   '/': explore,
+  '/explore': explore,
```

Comment:
Let's postpone this until we need it? For consistency, I suspect we should redirect / to /explore if we really want that route. Otherwise, it's a bit of a pain.